qa/tasks/vstart_runner.py

```diff
@@ -261,22 +261,29 @@ def close(self):
         return FakeStdIn(self)
 
 
-class LocalRemote(object):
+class LocalRemote(RemoteShell):
     """
     Amusingly named class to present the teuthology RemoteProcess interface when we are really
     running things locally for vstart
 
     Run this inside your src/ dir!
     """
 
-    os = Remote.os
-    arch = Remote.arch
-
     def __init__(self):
+        # NOTE: No need for calling RemoteSHell.__init__() since RemoteShell
+        # object required for teuthology requires much more details than
+        # LocalRemote's object would require.
```

Comment:
@batrick 

[`RemoteShell.__init__()`](https://github.com/ceph/teuthology/blob/master/teuthology/orchestra/remote.py#L335) contains plenty stuff that vstart_runner.py would never need to bother with. Is it really worth calling it?

The following works out fine when I tried it locally. Looks okay to you?

```suggestion
RemoteShell.__init__('local', None, None, None, None, None)
```

Reply:
Pushed this changes under a separate commit in case you wish to try it out.

Reply:
@rishabh-d-dave that's the `__init__` method of `Remote`, not `RemoteShell`. I don't see any `__init__` method for `RemoteShell`.

Reply:
Oh, I made a mistake there, `RemoteShell` doesn't inherit `Remote`, it's the other way round. Just `super(),__init__()` would be fine there. But still what's rationale behind it? I still find it redundant.

BTW, even `Remote` doesn't call `__init__()` of `RemoteShell`. Do we want that as well?

Reply:
Updated the DNM commit accordngly.